ssl/quic/quic_channel.c

```diff
@@ -1613,11 +1582,6 @@ static int ch_on_transport_params(const unsigned char *params,
             reason = TP_REASON_REQUIRED("ORIG_DCID");
             goto malformed;
         }
```

Comment:
Thinking more about it. It looks like our quic server does not include that `retry_source_connection_id` parameter after it sends its initial packet after validating client successfully. it feels like yet another piece which is missing in mosaic.

So I think this piece should also stay.

It looks like that once server creates channel after validating client, the server must remember to include this parameter to pass retry scid to client.